docs/docs/administer/configure.md

```diff
@@ -3,6 +3,8 @@ layout: default
 title: Configure
 nav_order: 1
 parent: Administer
+description: >
+  There are various environment variables that can be set to configure Firezone.
```

Comment:
clarification: Environment variables are something else, so maybe this could be "Configure Firezone by editing its main configuration file at /etc/firezone/firezone.rb."?